### ISSUE TITLE
Windows 10 v2004 "Graphics Setting" overrides user preference opencl of choice.

### ISSUE THREAD

**Issue by denchat** [qgis/QGIS#37396]
**Describe the bug**

OpenCL setting seems overriden by windows 10 graphic setting.

**How to Reproduce**

1. In OpenCL setting, select dedicated graphics. Close QGIS.
2. In Windows 10 setting / display / graphics settings.
3. Select integrated graphics for QGIS.
4. Start QGIS and OpenCL setting selection will follow one in Windows 10 graphics settings.
5. You can try changing but it keeps backing to windows 10 graphic settings.

**QGIS and OS versions**

QGIS version | 3.14.0-Pi
Compiled against GDAL/OGR | 3.0.4
Compiled against GEOS | 3.8.1-CAPI-1.13.3
Compiled against SQLite | 3.29.0
PostgreSQL Client Version | 11.5
QWT Version | 6.1.3 | 
Compiled against PROJ | 6.3.2
OS Version | Windows 10 (10.0)
QGIS code revision | 9f7028fd23
Running against Qt | 5.11.2
Running against GDAL/OGR | 3.0.4
Running against GEOS | 3.8.1-CAPI-1.13.3
Running against SQLite | 3.29.0
SpatiaLite Version | 4.3.0
QScintilla2 Version | 2.10.8
Running against PROJ | Rel. 6.3.2, May 1st, 2020

**Additional context**

  If this behaviour is enforced by windows itself, I guess windows user should be notified on the opencl settings.

**Comment by elpaso** [comment]
Is this a bug? If the OS in some way takes control of the hardware and switches the discrete GPU off,  QGIS cannot do anything about it.

**Comment by denchat** [comment]
I agree. Just adding some info for windows 10 v2004 user that they have to configure "Graphic Settings" in Display setting instead of this ui page is okey to me.

More verbose is to gray out the drop down list for windows 10 v2004 but I'm not sure it will express better perhaps make confusion that opencl not working.

**Comment by gioman** [comment]
@elpaso @denchat closing and reformulating this as feature request?

**Comment by denchat** [comment]
The new Nvidia driver do the same thing. Just leave the settings there and only poke a message with shortcut. <br/>

![nvidia](https://user-images.githubusercontent.com/19730041/85723909-e446c800-b71d-11ea-8878-4d609ea698a8.png)

**Comment by denchat** [comment]
@gioman Feature request is okey, I guess. The breaking change is on the OS side though.
